doc/authoring_command_modules/authoring_commands.md

```diff
@@ -7,15 +7,30 @@ The document provides instructions and guidelines on how to author individual co
 
 The basic process of adding commands is presented below, and elaborated upon later in this document.
 
-1. Write your command as a standard Python function.
-2. Register your command using the `cli_command` (or similar) function.
-3. Write up your command's help entry.
-4. Use the `register_cli_argument` function to add the following enhancements to your arguments, as needed:
+1. Create an __init__.py file for your command module.
+2. Write your command as a standard Python function.
+3. Register your command using the `cli_command` (or similar) function.
+4. Write up your command's help entry.
+5. Use the `register_cli_argument` function to add the following enhancements to your arguments, as needed:
     - option names, including short names
     - validators, actions or types
     - choice lists
     - completers
 
+**Create an __init__.py for your module**
```

Comment:
Shouldn't this be in the document for creating a new module rather than creating new commands?